src/plays/custom-memes-generator/CustomMemesGenerator.css

```diff
@@ -0,0 +1,185 @@
+.cmg-body {
```

Comment:
@Amanaakash Why don't you use this class `cmg-meme-main` as prefix for all the styles?

Reply:
i changed the styles to cmg-meme-main as their prefix